aws/environments/EXAMPLE.yml

```diff
@@ -3,17 +3,16 @@
 # The following variables should be put into a new yml file in this directory.
 # Any variables used by ansible roles may also be set here.
 
-region: us-west-2
+region: us-east-1
 
 # The {{ subdomain }}.{{ host_zone }} will be added to route53
 subdomain: dcoates.dev
 hosted_zone: lcip.org
 
-# the name of the ssl cert in IAM to use in the ELB
-# this must be compatible with your stack_name and hosted_zone
-# see http://docs.aws.amazon.com/IAM/latest/UserGuide/ManagingServerCerts.html
-ssl_certificate_name: exp20170412_wildcard_dev_lcip.org
-cdn_ssl_certificate_id: ASCAIAYDJPL5OT7LFEZSE
+# The ARN of the SSL certificate in ACM to use with Cloudfront & ELB
+# If you need to use us-west-2, use:
+# ssl_certificate_arn: arn:aws:acm:us-west-2:927034868273:certificate/ef6bb380-b02e-4433-9355-aec09dedd381
+ssl_certificate_arn: arn:aws:acm:us-east-1:927034868273:certificate/675e0ac8-23af-4153-8295-acb28ccc9f0f
```

Comment:
http://docs.aws.amazon.com/acm/latest/userguide/gs-cf.html
"3. ... To use an ACM certificate with CloudFront, you must request or import the certificate in the US East (N. Virginia) region."